alert-common/src/main/java/com/synopsys/integration/alert/common/channel/IssueTrackerChannel.java

```diff
@@ -82,8 +89,32 @@ public final void sendProviderCallbackEvents(List<ProviderCallbackEvent> callbac
     }
 
     private List<ProviderCallbackEvent> createCallbackEvents(IssueTrackerResponse issueTrackerResponse) {
-        // FIXME release an update to issuetracker-common
-        return List.of();
+        List<ProviderCallbackEvent> callbackEvents = new ArrayList<>();
+        for (IssueTrackerIssueResponseModel issueResponseModel : issueTrackerResponse.getUpdatedIssues()) {
+            AlertIssueOrigin alertIssueOrigin = issueResponseModel.getAlertIssueOrigin();
+
+            Optional<ComponentItem> optionalComponentItem = alertIssueOrigin.getComponentItem();
+            if (optionalComponentItem.isPresent()) {
+                ComponentItem componentItem = optionalComponentItem.get();
+                Optional<ComponentItemCallbackInfo> optionalCallbackInfo = componentItem.getCallbackInfo();
+                if (optionalCallbackInfo.isPresent()) {
```

Comment:
Can we just map here instead of 2 if statements?

Reply:
We need the component item later :/